tools/ovc/openvino/tools/ovc/convert_impl.py

```diff
@@ -500,6 +495,21 @@ def _convert(cli_parser: argparse.ArgumentParser, args, python_api_used):
                 print(ov_update_message)
 
         send_conversion_result('success')
+
+        if is_verbose(argv):
+            elapsed_time = datetime.datetime.now() - start_time
+            print('[ SUCCESS ] Total execution time: {:.2f} seconds. '.format(elapsed_time.total_seconds()))
+
+            # Statistics on allocated memory blocks per filename and per line number
+            snapshot = tracemalloc.take_snapshot()
+            stats = snapshot.statistics('lineno')
```

Comment:
I see that `tracemalloc` does not track memory allocated not in Python: https://stackoverflow.com/questions/50148554/when-would-the-python-tracemalloc-module-allocations-statistics-not-match-whats

Can you please check you implementation on some models and compare with ps stats?
Also, why don't you use `get_traced_memory` that reports peak memory consumption?

Reply:
> Also, why don't you use `get_traced_memory` that reports peak memory consumption?
Yes, we can use it. Done.

> Can you please check you implementation on some models and compare with ps stats?
Here are results of comparison with top command. 

fungi_V2 (from memory)
top: 2.1 GB
tracemalloc: 327.59 MB

fungi_V2 (from file)
top: 0.1 GB
tracemalloc: 0.08 MB. 

pytorch efficientnet_b7
tracemalloc: 8.03 MB
top: 0.5 GB

Results seem to correlate, but for all cases top shows significantly larger consumption. Probably it is due to c++ memory usage, but still difference looks big.